vnext/JSI/Shared/ChakraJsiRuntime_edgemode.cpp

```diff
@@ -1,40 +1,50 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-#include "ChakraRuntime.h"
+#include <ChakraRuntimeFactory.h>
+#include <SystemChakraRuntime.h>
 
 #include "ByteArrayBuffer.h"
 #include "Unicode.h"
 
-#if !defined(CHAKRACORE)
-#include <jsrt.h>
+// From <chakrart.h>
+STDAPI_(JsErrorCode)
+JsStartDebugging();
 
 namespace Microsoft::JSI {
 
-void ChakraRuntime::setupNativePromiseContinuation() noexcept {
+#if defined(USE_EDGEMODE_JSRT) && !defined(CHAKRACORE)
+/*static*/ void ChakraRuntime::initRuntimeVersion() noexcept {}
+#endif
+
+SystemChakraRuntime::SystemChakraRuntime(ChakraRuntimeArgs &&args) noexcept : ChakraRuntime(std::move(args)) {
+  this->Init();
+}
+
+void SystemChakraRuntime::setupNativePromiseContinuation() noexcept {
   // NOP
```

Comment:
>// NOP [](start = 2, length = 6)

Please note that it is not NOP in 0.64 where we did a number of changes to these classes to make sure that all JSI tests are passing. We need to be very careful moving these changes to newer RNW versions.

Reply:
Definitely. I will manually port this changes to master once this review has been approved, and do the normal backport cycle.

I started the review here to be able to validate Office in parallel.